### PR TITLE
feat(node): update to work with node 14

### LINKED ISSUE #16: unable to write HTML report in Node.JS 14.x
All tests run, but on completion an error is thrown:
```
Ran all test suites.
TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of XMLElement
    at Object.writeFileSync (fs.js:1380:5)
    at writeFile (.../node_modules/amex-jest-preset/html-report-creator.js:26:6)
```
I believe this is due to a breaking change in Node.JS 14.x where `data` is no longer being coerced https://github.com/nodejs/node/pull/31030

### PR DESCRIPTION
**Pull request by JamesSingleton**
Fixes #16

### CODE REVIEW

**Review by Francois-Esquire — DISMISSED** [review]

File: html-report-creator.js [hunk 1]
```diff
@@ -92,8 +92,12 @@ module.exports = (result) => {
       });
     }
   });
-  writeFile(process.env.JEST_TEST_REPORT_PATH || path.join(process.cwd(), 'test-results/test-report.html'), htmlOutput);
+  writeFile(process.env.JEST_TEST_REPORT_PATH || path.join(process.cwd(), 'test-results/test-report.html'), htmlOutput.end({
+    pretty: true,
+    indent: '  ',
+    newline: '\n',
+    allowEmpty: false,
+  }));
```
Comment (PixnBits) [inline html-report-creator.js]: given the length of the line, now that there's a function call with options maybe splitting the arguments into lines would be nicer? not a blocker
```suggestion
  writeFile(
    process.env.JEST_TEST_REPORT_PATH || path.join(process.cwd(), 'test-results/test-report.html'),
    htmlOutput.end({
      pretty: true,
      indent: '  ',
      newline: '\n',
      allowEmpty: false,
    })
  );
```

File: .gitignore [hunk 2]
```diff
@@ -9,6 +9,6 @@ jest-preset.json
 # misc
 .DS_Store
 npm-debug.log
-
+*.tgz
```
Comment (PixnBits) [inline .gitignore:14]: probably better to keep the newline between sections convention seen in the other sections
```suggestion
*.tgz

```

**Review by PixnBits — CHANGES_REQUESTED** [review]
thank you!
[edit]
I think it looks like Travis ran the tests on Node.JS 10.x but not 12.x or 14.x, should `.travis.yml` be updated to run the tests on multiple Node versions?

File: html-report-creator.js (hunk 1 above)
Comment (JamesSingleton) [reply html-report-creator.js]: Taken care of in 008ecd9

File: .gitignore (hunk 2 above)
Comment (JamesSingleton) [reply .gitignore:14]: Taken care of in 008ecd9

**Comment by JamesSingleton** [comment]
@PixnBits actually 8f647c3 fixes the travis config.

**Review by 10xLaCroixDrinker — APPROVED** [review]

**Review by PixnBits — APPROVED** [review]

**Pull request merged**